javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,36 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content, id });
+}
+
+function getNote(id) {
+  if (typeof id !== "number") {
+    console.log("Error: Invalid ID.");
+    return;
```

Comment:
You can write here "return undefined" for clarity
```suggestion
    return undefined;
```

And the error message can be made a little more specific (for example: **"ID must be a number"**).